test/integration/good/code-gen/cpp.expected

```diff
@@ -162,10 +162,14 @@ class eight_schools_ncp_model : public model_base_crtp<eight_schools_ncp_model>
     try {
       local_scalar_t__ mu;
       
+      current_statement__ = 1;
+      mu = in__.scalar();
       current_statement__ = 1;
       mu = in__.scalar();
```

Comment:
For some reason on your branch stanc3 is emitting twice as many data reads as it should... Maybe a bad merge?